zrml/prediction-markets/src/lib.rs

```diff
@@ -77,11 +72,7 @@ pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
 
     type Currency: ReservableCurrency<Self::AccountId>;
 
-    type Shares: Shares<Self::AccountId, Self::Hash, Balance = BalanceOf<Self>>
-        + WrapperShares<Self::AccountId, Self::Hash, Balance = BalanceOf<Self>>;
-
-    /// The identifier of individual markets.
-    type MarketId: AtLeast32Bit + Parameter + Member + MaybeSerializeDeserialize + Default + Copy;
```

Comment:
Can we keep the MarketId type for documentation purposed even though it's kind of useless in practice?

Reply:
No problem. Just a nit: `Asset` must have yet another type for `MarketId` but `MarketId` is not used everywhere like `Hash` so expect more trait associated types for `MarketId`.